core/templates/core/user_list.html

```diff
@@ -10,7 +10,7 @@ <h2>Welcome {{ user.get_username }}</h2>
             <img class="avatar" src="{{ user.socialaccount_set.all.0.get_avatar_url }}">
         {% endif %}
         <h4 class="create-deck-link">
-            <a href="{% url 'card_form' %}">Create Deck</a>
+            <a href="{% url 'deck_form' %}">Create Deck</a>  <a href="{% url 'card_form' %}">Create Card</a>
```

Comment:
need to add this to line 19 to show decks created by user:
```
<ul class="list-user-decks">
        {% for deck in decks %}
            {% if user == deck.creator %}
            <li class="user-deck-item"><a href="{{ deck.get_absolute_url }}">{{ deck.title }}</a></li>
            {% endif %}
        {% endfor %}
</ul>
 ```

Reply:
Okay, I updated the this PR as you suggested above. I also added some logic to correctly display 'Create Your Own Deck' if the user had not created any decks yet. In order to do so I had to pass a new 'user_created_decks' context from user_list_view.